### ISSUE TITLE
Message text overlaps timestamp

### ISSUE THREAD

**Issue by MrPeso4ko** [UnigramDev/Unigram#2530]
In some messages text overlaps time

Screenshot
![image](https://user-images.githubusercontent.com/75270212/134813960-c262f998-0a65-427e-aa21-3785a31b589e.png)

**Version Info**
 - Unigram 8.1 (6638) X64
 - Windows 20H2

Additional context
I use Windows 150% upscaling of interface

**Comment by FrayxRulez** [comment]
Any customization to the app/system font size?

**Comment by MrPeso4ko** [comment]
no, just what I stated in the issue, I use Windows 150% upscaling of interface

**Comment by FrayxRulez** [comment]
I haven't seen reports about this for a long time so I'll consider this fixed
